lib/fluent/plugin/out_mongo.rb

```diff
@@ -152,9 +147,14 @@ def emit(tag, es, chain)
     end
 
     def format(tag, time, record)
+      record = inject_values_to_record(tag, time, record)
       [time, record].to_msgpack
     end
 
+    def formatted_to_msgpack_binary
+      true
+    end
+
     def write(chunk)
       collection_name = @tag_mapped ? chunk.key : @collection
```

Comment:
How about deprecating `tag_mapped` parameter?
We can use `collection ${tag}` since v0.14.
So if the user specifies `tag_mapped` in configuration, we can change `collection` parameter to `${tag}` automatically.